src/main/java/org/spongepowered/common/command/sponge/SpongeCommand.java

```diff
@@ -191,6 +192,12 @@
             .addChild(reloadWorldCommand, "world")
             .build();
 
+        // /sponge uptime
+        final Command.Parameterized uptimeCommand = Command.builder()
+                .permission("sponge.command.uptime")
+                .shortDescription(Component.text("Shows Sponge JVM uptime."))
```

Comment:
nitpick: rather `Shows JVM uptime.` The property isn't anything specific to Sponge, just general JVM uptime.

Reply:
Fixed in 1435d1b